compose/bin/setup-integration-tests

```diff
@@ -4,15 +4,17 @@ source env/db.env
 MYSQL_INTEGRATION_CONFIG=dev/tests/integration/etc/install-config-mysql.php
 
 # If database doesn't exist create it and add user permissions
-bin/clinotty mysql -h${MYSQL_INTEGRATION_HOST} -uroot -p${MYSQL_ROOT_PASSWORD} ${MYSQL_INTEGRATION_DATABASE} -e exit &> /dev/null || 
+bin/clinotty mysql -h${MYSQL_INTEGRATION_HOST} -uroot -p${MYSQL_ROOT_PASSWORD} ${MYSQL_INTEGRATION_DATABASE} -e exit &> /dev/null ||
   bin/clinotty mysqladmin -h${MYSQL_INTEGRATION_HOST} -uroot -p${MYSQL_ROOT_PASSWORD} create ${MYSQL_INTEGRATION_DATABASE} &&
   echo "Database ${MYSQL_INTEGRATION_DATABASE} created." &&
   bin/cli mysql -uroot -p${MYSQL_ROOT_PASSWORD} -h${MYSQL_INTEGRATION_HOST} \
     -e "GRANT ALL PRIVILEGES ON ${MYSQL_INTEGRATION_DATABASE}.* TO '${MYSQL_INTEGRATION_USER}'@'%';FLUSH PRIVILEGES;"
 
-sed -e "s/'db-host' => 'localhost'/'db-host' => '${MYSQL_INTEGRATION_HOST}'/" \
-  -e "s/'db-password' => '123123q'/'db-password' => '${MYSQL_ROOT_PASSWORD}'/" \
-  -e "s/'amqp-host' => 'localhost'/'amqp-host' => 'rabbitmq'/" \
-  src/${MYSQL_INTEGRATION_CONFIG}.dist > src/${MYSQL_INTEGRATION_CONFIG}
+if [[ ! -f "src/${MYSQL_INTEGRATION_CONFIG}" ]]; then
+  MAGENTO_VERSION=$(bin/magento --version --no-ansi | cut -d" " -f 3)
```

Comment:
Should this be extracted to a separate script in preparation for #322 ?

Reply:
Perhaps, I'll merge this in though as it's quite old (didn't even see this PR)

Reply:
Agh, this is a very old commit that is much larger that I did not see. I think this needs to be rebased or merged with latest changes.

Reply:
@markshust my question was if we want to extract getting magento version to a separate script. At this stage it can be ignored until we know how we want to approach backwards compatibility for magento versions. If anything it can be done as a follow up PR.

Reply:
@markshust  Latest master is merged